reports/models.py

```diff
@@ -1 +1,29 @@
-# Create your models here.
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+
+
+@python_2_unicode_compatible
+class ReportTaskStatus(models.Model):
+    """ The status of the report generate tasks
+    """
+
+    STATUS_CHOICES = (
+        ('Pending', "Pending"),
+        ('Running', "Running"),
+        ('Sending', "Sending"),
+        ('Done', "Done")
```

Comment:
Thanks @rudigiesler I just copied a choice field from the registration models, it's better now and I also realized there was a missing choice with this implementation.

Ready for another look!